testing/component-testing-enzyme/src/__tests__/2-using-enzyme.test.js

```diff
@@ -116,31 +112,30 @@ test("Dialog renders button text when open and null when not open", () => {
 
 `📚 Exercise 4 - Testing more complex UI 📚
 
-Take a look at the SalesDialog below. Since we have 
-already tested the open and close functionality of the 
-Dialog, the IconButton, and the Icon individually, 
+Take a look at the SalesDialog below. Since we have
+already tested the open and close functionality of the
+Dialog, the IconButton, and the Icon individually,
 we only need to test what's unique about this particular component
 
 🛠️ Write a test for 3 things
 🛠️ 1) SalesDialog renders the copy within the P tag.
 🛠️ 2) Contains two Icon Buttons
 🛠️ 3) Each button renders the text passed to it
 
-You will need some new tricks for this - 
-you can just assert on the text of the entire Dialog component, 
+You will need some new tricks for this -
+if you just assert on the text of the entire Dialog component,
 as it will be a jumbled mess.
```

Comment:
This sentence also needs to be adjusted if you edited the text on the previous line.

Reply:
Oops, missed that.